### PR TITLE
feat(tracking): Add a new page_career_path_visited event

### PR DESCRIPTION
**Pull request by judahtanthony**
### Overview
Adding a new event with a category of page and a name of career_path_visited.

### PR Checklist
- [x] Related to JIRA ticket: REACH-513
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/tracking@0.4.0-alpha.0ea50e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://600075e03984be0ecd6e9e8a--gamut-preview.netlify.app>https://600075e03984be0ecd6e9e8a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/600075e03984be0ecd6e9e8a>Deploy Logs</a></p>

File: packages/tracking/src/types.ts [hunk 1]
```diff
@@ -221,3 +225,8 @@ export type UseUserResponse = {
   user?: User;
   status: string;
 };
+
+export declare type PagePathVisited = BaseEventData & {
```
Comment (dougyd92) [inline packages/tracking/src/types.ts:229]: nit: This type should be named `PagePathVisitedData`

**Review by dougyd92 — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/tracking@0.4.0-alpha.5a9680.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6000a05dc9b17144368abbb7--gamut-preview.netlify.app>https://6000a05dc9b17144368abbb7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6000a05dc9b17144368abbb7>Deploy Logs</a></p>

**Review by JoshuaKGoldberg — APPROVED** [review]

**Pull request merged**